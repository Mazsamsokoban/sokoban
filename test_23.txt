loadtest 23
showstate f1
showstate f2
showstate f3
move w right
showmaze