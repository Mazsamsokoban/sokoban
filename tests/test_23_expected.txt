f0:w f1:box1 f2:box2 f3:box3 f4:null 

name:f1
friction:Honey
thing:box1

name:f2
friction:Oil
thing:box2

name:f3
friction:Normal
thing:box3

f0:null f1:w f2:box1 f3:box2 f4:box3 

